### PR TITLE
Portfolio Desktop version

### PR DESCRIPTION
**Pull request by mohashyne**
# New Features added

- [ ] Add Desktop-ver header-hero
- [ ] Add Desktop-ver works/project section
- [ ] Add Desktop-ver skill/languages section
- [ ] Add Desktop-ver contact-form section

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -83,8 +95,8 @@ <h2 class="card-title">Tonic</h3>
                     </div>
                   </div>
                   
-                  <div class="grid-container card-2">
-                    <div class="card-container">
+             <div class="grid-container grid-container-desktop">
```
Comment (tufoinnkuo10) [inline dist/index.html:98]: - [ ] kindly note that while working in the desktop version, your mobile version should still look good like in the original design. please, kindly centralize your cards .
Screenshot (1055)_LI

File: dist/index.html [hunk 2]
```diff
@@ -30,41 +36,47 @@ <h1 class="lg-heading">
...
-            <section class="works-card">
-                <div class="grid-container card-1">
-                    <div class="card-container">
-                      <div>
-                        <img class="card-img" src="./img/projects/Snapshoot Portfolio_1.png" alt="photo of my 1st project" />
-                      </div>
-                         <div class="text-container">
-                           <h2 class="card-title">Tonic</h3>
+          <section class="works-card">
+              <div class="grid-container grid-container-desktop">
```
Comment (tufoinnkuo10) [inline dist/index.html:73]: - [ ] Great job with the cards, please kindly place the text beside the image well and ensure the heading is almost aligned with the image at the top like in the original design.
yours

Screenshot (1054)_LI
figma
Screenshot (1053)

File: dist/index.html [hunk 3]
```diff
@@ -8,17 +8,23 @@
...
+    <div class="container desk-container">
         <header class="menu">
             <a href="#" class="logo">Crystal logo</a>
             <nav class="burger">
               <span></span>
               <span></span>
               <span></span>
             </nav>
+            <nav class="desktop-menu">
+              <a href="#">Portfolio</a>
```
Comment (tufoinnkuo10) [inline dist/index.html:20]: - [ ] please note that the headline section of the desktop version is not centralized. It is aligned to the left. Kindly fix yours so it follows the original design.
- [ ] kindly follow the font sizes and weight as well

yours
Screenshot (1049)
figma
Screenshot (1050)

File: dist/css/style.css [hunk 4]
```diff
@@ -632,3 +633,231 @@ h1 {
...
+
+} 
+
+/* Contact form */
+
+#contact-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 130%;
```
Comment (tufoinnkuo10) [inline dist/css/style.css:841]: - [ ] please set this width to 100% as well

File: dist/css/style.css [hunk 5]
```diff
@@ -632,3 +633,231 @@ h1 {
...
+}
+
+/* Works & Project */
+.desktop-about-skills {
+  display: grid;
+  grid-template-rows: auto auto;
+  background-color: #fff;
+  border-top-right-radius: 80px;
+  padding: 114px 24px;
+  width: 130%;
```
Comment (tufoinnkuo10) [inline dist/css/style.css:716]: - [ ] kindly set this width to 100% to avoid the horizontal scroll on the page

File: dist/index.html [hunk 6]
```diff
@@ -157,88 +169,91 @@ <h2 class="card-title">Multi-Post <br> Stories</h3>
...
+                  <img src="./img/icons/micro.svg"/>
+                </a>
+                </li>
+               <li>
+              <a href="#" class="icons"><img src="./img/icons/git.svg"/></a>
+               </li>
+               <li>
+                 <a href="#" class="icons"><img src="./img/icons/hand.svg" /></a>
+               </li>
+             </ul>
```
Comment (tufoinnkuo10) [inline dist/index.html:204]: - [ ] please kindly reduce the gap between the about info and the languages so it matches the original design
yours
Screenshot (1051)_LI
original
Screenshot (1052)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @mohashyne 

nice work so far. There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Figma design well followed

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: dist/index.html (hunk 2 above)
Comment (Qoosim) [inline dist/index.html:73]: - [ ] Great work implementing required changes from the previous reviewer. However, there're few things you still need to work on before you move on to the next milestone. But you're almost there.
When you look at the Figma design very closely, you would see that both the image part and the image info share the card with equally plus the space between them. But your design looks a bit different from Figma design. Kindly look into that and fix.

### Your Design
your-project-card

### Figma Design
temp-project-card

File: dist/index.html [hunk 7]
```diff
@@ -157,88 +169,91 @@ <h2 class="card-title">Multi-Post <br> Stories</h3>
...
+                 <a href="#" class="icons">
+                  <img src="./img/icons/micro.svg"/>
+                </a>
+                </li>
+               <li>
+              <a href="#" class="icons"><img src="./img/icons/git.svg"/></a>
+               </li>
+               <li>
+                 <a href="#" class="icons"><img src="./img/icons/hand.svg" /></a>
+               </li>
```
Comment (Qoosim) [inline dist/index.html:203]: - [ ] As suggested by the previous reviewer, There's still more than required space between `About Myself` and the `Language` that need to be reduced. Kindly work on that too.

### Your Design
your-about-spacing

### Figma Design
temp-about-spacing

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @mohashyne, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GiHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 2 above)
Comment (usorfaitheloho) [reply dist/index.html:73]: You did really well working on the changes by the previous reviewer, you're really close to moving to the next project but you have just some few changes 😄 

You've already adjusted the margin between the elements, how about you reduce the padding between them as it is on the high side at the moment to enable us to have a perfect replica of the design and please also remeber that `Back End Dev` is on one line and the font size used for that section is a bit bigger than what you have 

your design:

image

Figma design :

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @mohashyne  :thumbsup:  ,

This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are still some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good use of JavaScript, HTML&CSS best practices ✔️ 
-  Well-structured files ✔️
<br/>
<br/>
<br/>

 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @mohashyne  , 
# STATUS: APPROVED! 🥇 💯 
- [x] Very Nice Design, Keep it up!!
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**